Sources/CardPayments/Models/ServiceCard.swift

```diff
@@ -0,0 +1,79 @@
+import Foundation
+
+/// ServiceCard is used for formatting contents of `CardRequest` for `ConfirmPaymentSourceRequest`
+struct ServiceCard: Encodable {
```

Comment:
The Card struct with Encodable was used to get Card info from the customer and also included as part of  CardRequest which now accepts optional vault object to form a request body for confirm-payment-source endpoint. 

The Card struct to collect data has no need for attribute section. I created ServiceCard to be able to format the request for the endpoint.

Reply:
Another option is to leverage Swift's namespacing to create an internal `Card` type, and it will not conflict with the public facing one. Here's a branch with a simplified diff example off of `main` --> https://github.com/paypal/iOS-SDK/compare/demo-internal-card-type. The internal type in the example would be accessed like `ConfirmPaymentSource.PaymentSource.Card` due to the way it's nested. We can flatted things as well with custom encoders if you want, but IMO this route if using Swift namespacing is easier to understand than another `Service` layer.

Thoughts? cc: @jaxdesmarais

Reply:
I like that. I was considering putting the ServiceCard within the `ConfirmPaymentRequest` struct. It makes more sense the way you have it. I thought having another Card struct might be confusing. I like the way you can specify within the PaymentSource struct itself `PaymentSource.Card`, I didn't know that.

Reply:
I dig this approach as well. I think it achieves @KunJeongPark goal of moving the API layer out of the public facing interface while also keeping our interface cleaner and more maintainable vs creating a `Service` layer. 🚀

Reply:
I put the new private Card struct at the ConfirmPaymentSourceRequest level since I got "at most 1 level deep" warning when I embedded it in the PaymentSource.